packages/transaction-manager/examples/chained-transfer-example/src/index.ts

```diff
@@ -0,0 +1,102 @@
+import { Cell, Hash, Script, blockchain } from '@ckb-lumos/base';
+import { BI } from '@ckb-lumos/bi';
+import { Indexer } from '@ckb-lumos/ckb-indexer';
+import { initializeConfig, predefined } from '@ckb-lumos/config-manager/lib';
+import { TransactionSkeleton, TransactionSkeletonType, encodeToAddress, sealTransaction } from '@ckb-lumos/helpers';
+import { common } from '@ckb-lumos/common-scripts';
+import { bytes } from '@ckb-lumos/codec';
+import { key } from '@ckb-lumos/hd';
+import { TransactionManager } from '@ckb-lumos/transaction-manager';
+
+const ALICE_PRIVATE_KEY = '0x53815fbee34af63e686f5cad7db8074b4b8fd4473617dee2db0ae84d2c6325c4';
+const ALICE_ARGS = '0xe9441c447677de3f24fc64ce03f46fd259ed5e8c';
+const RPC_URL = 'https://testnet.ckb.dev';
+const _100_CKB_SHANNON = 10_000_000_000;
+const TX_FEE_SHANNON = 100_000;
+const CONFIG = predefined.AGGRON4;
+initializeConfig(CONFIG);
+
+const indexer = new Indexer(RPC_URL);
+const SECP256K1_BLAKE160 = CONFIG.SCRIPTS.SECP256K1_BLAKE160!;
+const aliceLock: Script = {
+  codeHash: SECP256K1_BLAKE160.CODE_HASH,
+  hashType: SECP256K1_BLAKE160.HASH_TYPE,
+  args: ALICE_ARGS,
+};
+const aliceAddress = encodeToAddress(aliceLock);
+const txManager = new TransactionManager({
+  providers: {
+    rpcUrl: RPC_URL,
+  },
+});
+
+/**
+ * @description This example shows how to use transaction manager to collect cells and send a transaction.
+ */
+async function main() {
+  const txHash1 = await transferViaTxManager();
+  const txHash2 = await transferViaTxManager();
+  console.log('txHash1:', txHash1);
+  console.log('txHash2:', txHash2);
+}
+
+main();
+
+
+async function transferViaTxManager(): Promise<Hash> {
+  const cells = await collectCells();
```

Comment:
maybe we can just use `common.transfer` to avoid making the example too complex


```ts
const manager = new TransactionManager();
let txSkeleton = await common.transfer(TransactionSkeleton({ cellProvider: manager }), ...);
// ...
```

Reply:
Tried to use `common.transfer` but just returns empty tx, maybe wrong usage, I'll look into the source code of `common.transfer` and try again.

Reply:
Another reason why fail to use `common.transfer` is that `TransactionManager.collector` reutrns `Promise<CellCollector>` which is not compatible to `CellCollector`, because `TransactionManager` needs to read from `storage`, which is async storage, so it returns a promise